### PR TITLE
Instructor notes

### PR DESCRIPTION
**Pull request by jessicamizzi**
This PR adds instructor notes for the Wrangling Genomics lesson. It covers section 00-quality-control, 01-trimming, 02-variant_calling, but does not cover 03-variant calling.

### CODE REVIEW

**Comment by raynamharris** [comment]
address issue https://github.com/data-lessons/wrangling-genomics/pull/14

File: _extras/guide.md [hunk 1]
```diff
@@ -3,4 +3,101 @@ layout: page
...
+>                ${base}_2.trim.fastq.gz ${base}_2un.trim.fastq.gz \
+>                SLIDINGWINDOW:4:20 MINLEN:25 ILLUMINACLIP:NexteraPE-PE.fa:2:40:15 
+> done
+~~~
+
+#### bwa mem Example Command
+The first bwa mem command is an example and is not meant to be run. It is:
+
+~~~
+bwa mem ref_genome.fasta input_file_R1.fastq input_file_R2.fastq > output.sam
```
Comment (raynamharris) [inline _extras/guide.md]: would be good to add . `#` at the beginning to indicate that this should not be run

File: _extras/guide.md [hunk 2]
```diff
@@ -3,4 +3,101 @@ layout: page
...
+~~~
+
+The correct command follows:
+
+~~~
+bwa mem data/ref_genome/ecoli_rel606.fasta data/trimmed_fastq_small/SRR2584866_1.trim.sub.fastq data/trimmed_fastq_small/SRR2584866_2.trim.sub.fastq > results/sam/SRR2584866.aligned.sam
```
Comment (raynamharris) [inline _extras/guide.md]: would be good to add a `$` to the beginning.

**Comment by jessicamizzi** [comment]
thanks, @raynamharris, I added a # and $ to denote a comment and code to be run.

**Pull request merged**